package.json

```diff
@@ -9,11 +9,13 @@
     "lint": "eslint --ext js ./",
     "lint:fix": "npm run lint -- --fix",
     "prettier": "prettier --list-different './**/*.{js,jsx}'",
-    "prettier-clean": "prettier --write './**/*.{js,jsx}'"
+    "prettier-clean": "prettier --write './**/*.{js,jsx}'",
+    "cypress": "cypress open"
```

Comment:
Super useful to have the script written and also in the README.md, thanks!!

Reply:
Hooray for less typing!